src/query/sql/src/planner/planner.rs

```diff
@@ -67,8 +70,31 @@ impl Planner {
     pub async fn plan_sql(&mut self, sql: &str) -> Result<(Plan, PlanExtras)> {
         let settings = self.ctx.get_settings();
         let sql_dialect = settings.get_sql_dialect()?;
+        // compile prql to sql for prql dialect
+        let mut prql_converted = false;
+        let final_sql: String = match sql_dialect == Dialect::PRQL {
+            true => {
+                let options = prqlc::Options::default();
+                match prqlc::compile(sql, &options) {
+                    Ok(res) => {
+                        info!("Try convert prql to sql succeed, generated sql: {}", &res);
+                        prql_converted = true;
+                        res
+                    }
+                    Err(e) => {
+                        warn!(
```

Comment:
So each time we turn back into the raw SQL, there will be warn msg.

```
set sql_dialect = 'prql';
...

set sql_dialect = 'mysql';  ----  we should not print this msg I think.
```

Reply:
Yes, SetVariable statement still work at prql dialect, just print a warn log.
We cannot check sql statement type before parse, so we cannot check wheater we should print warn log.